spec/controllers/api/v1/token_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+require  "#{Rails.root}/spec/support/controller_helpers.rb"
```

Comment:
rspec already has a config to load files under spec/support, check the spec_helper it might be commented